test/framework/kogitobuild.go

```diff
@@ -60,6 +62,13 @@ func cliDeployKogitoBuild(buildHolder *bddtypes.KogitoBuildHolder) error {
 		cmd = append(cmd, gitURI)
 	}
 
+	// webhooks
+	if len(buildHolder.Spec.WebHooks) > 0 {
+		for _, webhook := range buildHolder.Spec.WebHooks {
+			cmd = append(cmd, "--web-hook", fmt.Sprintf("%s=%s", webhook.Type, webhook.Secret))
```

Comment:
Can you move the code to the mappers.GetBuildCLIFlags among other KogitoBuild flags?

Reply:
Moved